### ISSUE TITLE
A separate view for blocking/blocked queries

### ISSUE THREAD

**Issue by kmoppel** [zalando/pg_view#26]
Would be nice to have something like "Waiting/Blocking" queries screens in [pg_activity](https://github.com/julmon/pg_activity) but on one page and as a tree maybe. Asides from that a TPS counter would also be nice :)

**Comment by kmoppel** [comment]
Also an idea would be to have a possibility to show only queries from a specific (list/pattern) of user, and then also other queries that are being blocked by this user.

**Comment by LappleApple** [comment]
@kmoppel @alexeyklyukin Same question as for Issue #12. We could re-tag as "Help Wanted" and try to pitch these enhancements as contribution opportunities  ...
